conf/modules.config

```diff
@@ -589,6 +589,40 @@ process {
         ]
     }
 
+    //
+    //  BEDTOOLS_COVERAGE
+    //
+    withName: SAMTOOLS_VIEW_GENOME {
+        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
+        ext.args = '-H --output-fmt sam'
+        ext.prefix = "genome"
+        ext.args2 = "&& cat genome.sam |  grep '@SQ' | sed 's#@SQ\tSN:\\|LN:##g' > genome.txt && mv genome.txt genome.sam"
+        publishDir = [
+            enabled: false
+        ]
+    }
+    withName: BEDTOOLS_COVERAGE_MEAN {
+        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
+        ext.args = '-mean -nonamecheck'
+        ext.prefix = { "${meta.sample_id}_${meta.library_id}_${meta.reference}_depth" }
+        publishDir = [
+            path: { "${params.outdir}/bedtools/" },
+            mode: params.publish_dir_mode
+        ]
+        cache = true
+    }
+    withName: BEDTOOLS_COVERAGE {
+        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
+        ext.args = '-nonamecheck'
+        ext.prefix = { "${meta.sample_id}_${meta.library_id}_${meta.reference}_breadth" }
+        publishDir = [
+            path: { "${params.outdir}/bedtools/" },
+            mode: params.publish_dir_mode
+        ]
+        cache = true
```

Comment:
Is ther a reason why this is necessary?

Reply:
I noticed when i was testing this it wouldn't cache with -resume if it had already run. I'll look into it a bit more.

Reply:
If the file was remote (i.e., via URL ) cacheing may not work